### ISSUE TITLE
Enhancement Read Only as Form Static Inputs

### ISSUE THREAD

**Issue by rico** [kristijanhusak/laravel-form-builder#121]
Thanks for the package - love it so far. What I really would like to have, is a simple way to display a form with `static` inputs only (especially for the `show` method of a `Route::resource`).

I know that you just implemented `disableFields()` method. But in my opinion, `static` Bootstrap fields looks just better than `readonly` inputs.

**Comment by kristijanhusak** [comment]
I agree, but i didn't implement it (yet), because i don't know how show some fields as static, for example list of checkboxes (choice multiple expanded)  or multiple select. Do you have any idea?

**Comment by rico** [comment]
Hi, thank you for the quick reply - and sorry it took me so long. However, happy to hear you agree on this :-)

I would suggest, for example, using **Glyphicons** for checkboxes (`glyphicon-unchecked`, `glyphicon-check`), together with **List groups** or normal **Unordered lists** to display checkboxes or multiple selections. I used such combinations in other projects and it looked quite good - imho.

I would do it as simple as possible, but give the user the ability to change the view in a custom template or templates (like the templates for the different inputs).

**Comment by kristijanhusak** [comment]
@rico 
I'm not sure using glyphicons is good idea, because in that way i'm tight coupling the views to bootstrap only. 
Also, I'm thinking of creating separate class for this read only functionality, where you would pass form instance to it to get that behavior. I don't want to add too much responsibilites to the Form class, it has many already.

**Comment by rico** [comment]
@kristijanhusak - can't you couple the static views to the `FormBuilder` (or some other class) as you did with the existing inputs? If one uses your package out of the box, they are all coupled to Bootstrap by *default* anyway.

I completely understand your concern about not adding to much responsibilities to the Form class. However, from a developers perspective, I think it would be much more convenient to have a method on the Form class itself. Maybe you could separate the logic using a `trait`?
